classes/controller/AdminController.php

```diff
@@ -313,9 +313,14 @@ public function __construct()
 		if ($this->multishop_context == -1)
 			$this->multishop_context = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
 
-		$this->bo_theme = ((Validate::isLoadedObject($this->context->employee) && $this->context->employee->bo_theme) ? $this->context->employee->bo_theme : 'default');
+        // Fix #PSCSX-4339 - "default" theme name should be customizable
```

Comment:
Sure, sorry for that, i'm using a temporary laptop
i'm not on my desktop.

On 15/01/2015 17:58, Gregory Roussac wrote:
> In classes/controller/AdminController.php
> <https://github.com/PrestaShop/PrestaShop/pull/2379#discussion-diff-23024493>:
>
>> @@ -313,9 +313,14 @@ public function __construct()
>>  		if ($this->multishop_context == -1)
>>  			$this->multishop_context = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
>>
>> -		$this->bo_theme = ((Validate::isLoadedObject($this->context->employee) && $this->context->employee->bo_theme) ? $this->context->employee->bo_theme : 'default');
>> +        // Fix #PSCSX-4339 - "default" theme name should be customizable
>
> Hi,
>
> thank you for your PRs.
> Can you please use tabs as indentation ?
>
> Regards
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PrestaShop/PrestaShop/pull/2379/files#r23024493>.
>